ui/settings.glade

```diff
@@ -1158,7 +1170,7 @@ If the file was not yet saved you can find it in ~/.cache/xournalpp/autosaves&lt
                                             <property name="visible">True</property>
                                             <property name="can-focus">False</property>
                                             <property name="tooltip-text" translatable="yes">Gaussian parameter: the higher, the more stabilization and the more lag between your cursor and the painting tip.</property>
-                                            <property name="label" translatable="no">σ</property>
```

Comment:
Did you change this on purpose?

Reply:
Oops! I must have accidentally pressed something within the Glade UI - I'll check the diff next time so this doesn't sneak in again...